lib/scss/components/cta-list/_cta-list.scss

```
@@ -0,0 +1,112 @@
+@charset "UTF-8";
+
+.su-cta-list-paragraph{

```

```suggestion
.su-cta-list-paragraph {
```

Should have a space between statement and brace.